platform/core/src/services/UICustomizationService/types.ts

```diff
@@ -0,0 +1,33 @@
+import Command from '../../types/Command';
+
+export interface BaseUICustomization extends Record<string, unknown> {
+  id: string;
+  customizationType?: string;
+  description?: string;
+  label?: string;
+  commands?: Command[];
+}
+
+export interface LabelUICustomization extends BaseUICustomization {
```

Comment:
I see the BaseUICustomization has `label` why LabelUICustomization is adding it as well?